skfda/datasets/_real_datasets.py

```diff
@@ -1206,3 +1206,95 @@ def fetch_gait(
 
 if fetch_gait.__doc__ is not None:  # docstrings can be stripped off
     fetch_gait.__doc__ += _gait_descr + _param_descr
+
+_handwrit_descr = """
+    Data representing the X-Y coordinates of 20 writing the "fda".
+
+    References:
+        Ramsay, James O., and Silverman, Bernard W. (2006),
+        Functional Data Analysis, 2nd ed. , Springer, New York.
+"""
+
+
+@overload
+def fetch_handwrit(
```

Comment:
Please, use a different name for this dataset. Partial names are confusing.

Reply:
the name in fda is handwrit

Reply:
I know it. But we do not need to use the same name.

Reply:
ok, handwritten? signature?

Reply:
fda_handwrit?

Reply:
`handwriting`. Also, Ramsay's book has a more thorough description of the dataset.